### PR TITLE
Standardizing Windows user data path

### PR DESCRIPTION
**Pull request by raph-luc**
Minor change

First described here https://github.com/ansys/pyfluent/issues/1588, and later https://github.com/ansys/pyprimemesh/issues/484

PRs: https://github.com/ansys/pyfluent/pull/1615, https://github.com/ansys/pymapdl/pull/2064, https://github.com/ansys/pyprimemesh/pull/485

1. Changes `SETTINGS_DIR` from `user_data_dir("ansys_tools_path")` which resolved to `%LocalAppData%\ansys_tools_path\ansys_tools_path` on Windows, to `user_data_dir(appname="ansys_tools_path", appauthor="Ansys")` which resolves to `%LocalAppData%\Ansys\ansys_tools_path`, same naming scheme as in PyFluent and the default Ansys Fluent Windows installation already creates the `%LocalAppData%\Ansys` root folder. <br> <br> This does not change anything in Linux as `appauthor` is unused on Linux. 
3. Exposing `SETTINGS_DIR` so that it can be used as `ansys.tools.path.SETTINGS_DIR` rather than `ansys.tools.path.path.SETTINGS_DIR` 
4. Reworked the examples for functions `save_mapdl_path` and `save_mechanical_path` to reflect these changes (no changes to the actual functions)

Note/food for thought: when `ansys.tools.path` is first imported, regardless of what the user is going to do, `SETTINGS_DIR` folder is immediately created, even if it is not going to be used. I would think this is probably unnecessary and you might want to change this (could break CI/tests though).

### CODE REVIEW

**Review by RobPasMue — APPROVED** [review]
LGTM - But I'll wait for @germa89 to review

File: src/ansys/tools/path/path.py [hunk 1]
```diff
@@ -570,15 +570,15 @@ def save_mapdl_path(exe_loc=None, allow_prompt=True) -> str:
...
 
     .. code:: pycon
 
-        >>> import appdirs
+        >>> from ansys.tools.path import SETTINGS_DIR
         >>> import os
-        >>> print(os.path.join(appdirs.user_data_dir("ansys_tools_path"), "config.txt"))
-        C:/Users/user/AppData/Local/ansys_tools_path/ansys_tools_path/config.txt
+        >>> print(os.path.join(SETTINGS_DIR, "config.txt"))
+        C:/Users/[username]]/AppData/Local/Ansys/ansys_tools_path/config.txt
```
Comment (germa89) [inline src/ansys/tools/path/path.py:581]:
```suggestion
        C:/Users/[username]/AppData/Local/Ansys/ansys_tools_path/config.txt
```

**Review by germa89 — APPROVED** [review]
LGTM. Just a typo.

File: src/ansys/tools/path/path.py [hunk 2]
```diff
@@ -570,15 +570,15 @@ def save_mapdl_path(exe_loc=None, allow_prompt=True) -> str:
 
     Notes
     -----
-    The configuration file location (``config.txt``) can be found in
-    ``appdirs.user_data_dir("ansys_tools_path")``. For example:
```
Comment (koubaa) [inline src/ansys/tools/path/path.py:574]: @germa89 Pre-existing - but this helpstring does not quite match the purpose of `save_mapdl_path`

**Comment by koubaa** [comment]
@raph-luc @germa89  I'm concerned with the lack of migration here.  If the user configured pymapdl a certain way, by upgrading to a new ansys-tools-path, the config file would be read from somewhere else and their configuration is lost.  This isn't what I'd expect from a minor version upgrade.

Was this already released?

**Pull request merged**